### ISSUE TITLE
integration with iPlant

### ISSUE THREAD

**Issue by bobthacker** [arborworkflows/aRbor#48]
Users requested the ability to integrate with their iPlant accounts.
Hosting a version of aRbor at iPlant/atmosphere could allow:
- user authentication based on existing iPlant users
- file directory system so that existing iPlant users can pull their data into iPlant.

**Comment by curtislisle** [comment]
Agreed this is a good deployment option.  I want to defer detailed integration until we are in the extended Arbor project:  after our core project finishes and we are on the "software deployment" phase of the project.  Practically it means this would be late Summer 2015.  If this time is too late, we can decide what to push back instead.  Deploying Arbor on Amazon cloud is pretty straightforward.  A partial integration with atmosphere deployment might be easier.  Let's talk more about this.
